app/src/views/features/sessions/SessionsIndexPageContainer/SessionsIndexPage/RecordingsList/index.js

```diff
@@ -60,9 +61,28 @@ const RecordingsList = ({
         width: "30%",
         render: (name, record) => {
           return (
-            <Link to={PATHS.SESSIONS.SAVED.ABSOLUTE + "/" + record.id} state={{ fromApp: true }}>
-              {name}
-            </Link>
+            <div className="display-flex">
+              <Link
+                to={
+                  record.isDraft
+                    ? `${PATHS.SESSIONS.DRAFT.ABSOLUTE}/${record.id}?saved`
+                    : `${PATHS.SESSIONS.SAVED.ABSOLUTE}/${record.id}`
+                }
+                state={{ fromApp: true }}
+              >
+                {name}
+              </Link>
+              {record.isDraft && (
+                <span style={{ marginLeft: "8px" }}>
+                  <InfoTag
+                    title="DRAFT"
+                    description="Only last 3 drafts are saved in browser cache. Please save these recording so that you don't lose on the
+        data."
```

Comment:
```suggestion
                    description="Only last 3 drafts are saved in browser cache. Please save these recording so that you don't lose the data."
```